recipes/resiprocate/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,71 @@
+#include <resip/stack/SipMessage.hxx>
```

Comment:
@AlexeiKislinskii10 Is there a way you can add another test_package similar to https://github.com/uilianries/conan-resiprocate/tree/release/1.10.2/test_package ?

Reply:
Thanks a noteworthy suggestion 👀 

The goal for CCI is to ensure the package is correct (headers/macros are present and links to libs) so we like to avoid the overly complicated testing library or example code... that's the projects responsibility.

Good thought 🥇  just not required here 😃

Reply:
Actually I can, with minimum of changes, just copy this test in new version of reSIProcate package, but is it needed? Test package should be simple to explain how to use this lib from conan.
@prince-chrismc is there any topic with rules about test package? Because I found only [this](https://docs.conan.io/en/latest/creating_packages/getting_started.html#the-test-package-folder) note